k256/src/arithmetic/affine_op.rs

```diff
@@ -0,0 +1,342 @@
+use crate::AffinePoint;
+use crate::arithmetic::FieldElement;
+use crate::arithmetic::mul::{
+    Endomorphism, Identity, LookupTable, Radix16Decomposition, lincomb as lincomb_pippenger,
```

Comment:
```suggestion
    Endomorphism, Identity, LookupTable, Radix16Decomposition, lincomb,
```
Or is there a reason to rename it?

Reply:
yes, there is already a lincomb function defined.   `lincomb` is the function name belongs to `LinearCombination` trait, it initialises some values and then call another lincomb function, which is the generalised lincomb. In PowdrAffinePoint implementation, I also create a lincomb function (standalone) to initialise some values and then call the generalised lincomb, so need to rename it here due to conflict.

Reply:
Right, I see. You could move `lincomb` into the `impl PowdrAffinePoint {` block. Then it would be an associated function and be called as `PowdrAffinePoint::lincomb`, similar to how `ProjectivePoint::lincomb` is called.

Reply:
Done. Thanks!